src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string;
 
-}
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
+
+    function properTriangle(
+      aSide: number,
+      bSide: number,
+      cSide: number,
+    ): boolean {
+      const allSides = [aSide, bSide, cSide];
+      const longgestSide = Math.max(aSide, bSide, cSide);
+      const sumOfshorterSides = allSides
+        .filter((side) => side !== longgestSide)
+        .reduce((total, side) => total + side);
 
-export class Circle {
+      return longgestSide < sumOfshorterSides;
+    }
 
+    if (this.a <= 0
+      || this.b <= 0
+      || this.c <= 0
+      || !properTriangle(this.a, this.b, this.c)
```

Comment:
you should move all the conditions to the new line if you move at least one
```suggestion
    if (
      this.a <= 0
      || this.b <= 0
      || this.c <= 0
      || !properTriangle(this.a, this.b, this.c)
```